pkg/apis/core/types.go

```diff
@@ -3960,16 +3960,32 @@ type PodStatus struct {
 	// +optional
 	QOSClass PodQOSClass
 
-	// The list has one entry per init container in the manifest. The most recent successful
+	// Statuses of init containers in this pod. The most recent successful non-restartable
 	// init container will have ready = true, the most recently started container will have
 	// startTime set.
+	// The list must not contain entries with the same container name, and must only contain
```

Comment:
If it's not enforced, then the right word is `should`

Reply:
fixed